.github/workflows/build-lint-test.yml

```diff
@@ -192,11 +192,16 @@ jobs:
         if: ${{ matrix.package-name == '@metamask/snaps-controllers' || matrix.package-name == '@metamask/snaps-execution-environments' || matrix.package-name == '@metamask/snaps-utils' }}
         uses: MetaMask/action-retry-command@v1
         with:
-          command: yarn install-chrome
+          command: yarn playwright install --with-deps --only-shell chromium firefox
```

Comment:
How will this work locally? Do you need to run any commands besides `yarn` and `yarn test` to run browser tests locally?

Reply:
Yes, Vitest will tell you to run `yarn playwright install` on the first time running `yarn test`.